### DISCUSSION TITLE
Transient fix

### DISCUSSION THREAD

**Discussion by anegg0** [magit/transient#105] [Support] [answered]
Hi, after updating my spacemacs to 27, magit stopped working (no access to push or commit) and I tried (in vain) any solution. Until I discovered that the MELPA repo has two versions of transient. I fixed my problem by installing the latest one. I hope it helps.

**Comment by tarsius** [comment, marked as answer]
> Hi, after updating my spacemacs to 27, magit stopped working (no access to push or commit) and I tried (in vain) any solution.

Apparently there was some breaking change that made it necessary to update.

> Until I discovered that the MELPA repo has two versions of transient.

Melpa only ever has one version of a package. Your local package list features two versions though: the one you had currently installed and the one that was available for installation from Melpa.

> I fixed my problem by installing the latest one.

But regardless of the previous point--this is the proper fix.

**Reply by anegg0** [reply]
thanks for clarifying, @tarsius :)
